tests/e2e/backup_restore_suite_test.go

```diff
@@ -199,7 +199,7 @@ func runBackup(brCase BackupRestoreCase, backupName string) bool {
 	gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
 	// wait for backup to not be running
-	gomega.Eventually(lib.IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*20, time.Second*10).Should(gomega.BeTrue())
+	gomega.Eventually(lib.IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*60, time.Second*10).Should(gomega.BeTrue())
```

Comment:
I think this is not ideal

would run in prow sometimes and see how long it takes, and put a little above

but to much above can hunt us in the future, this is also used by non VM tests

Reply:
I added per-test timeouts, if that's any better. When I run E2E tests locally I see backups take 30+ minutes for the Fedora VM.

Reply:
oh right, good catch